usersync/chooser_test.go

```diff
@@ -373,9 +381,10 @@ func (fakeSyncer) GetSync(syncTypes []SyncType, privacyPolicies privacy.Policies
 }
 
 type fakePrivacy struct {
-	gdprAllowsHostCookie bool
-	gdprAllowsBidderSync bool
-	ccpaAllowsBidderSync bool
+	gdprAllowsHostCookie   bool
+	gdprAllowsBidderSync   bool
+	ccpaAllowsBidderSync   bool
+	activityAllowIUserSync privacy.ActivityResult
```

Comment:
Is there a typo here with `activityAllowIUserSync`? Should it be `activityAllowsUserSync`?

Reply:
This is typo. Fixed